cashctrl_ledger/constants.py

```diff
@@ -7,3 +7,9 @@
     "credit": "float64",
     "taxName": "string[python]",
 }
+
+FX_REVALUATION_ACCOUNT_COLUMNS = {
+    "foreign_currency_account": "int",
+    "fx_gain_loss_account": "int",
+    "exchange_rate": "float64",
```

Comment:
I suggest to use `Float64` (with uppercase F) to explicitly indicate that a column can contain missing values.

```python
>>> df = pd.DataFrame({'a': [34.5, None], 'b':[34.5, None]})
>>> df['a'] = df['a'].astype('float')
>>> df['b'] = df['b'].astype(pd.Float64Dtype())
>>> df
      a     b
0  34.5  34.5
1   NaN  <NA>
>>> df.dtypes
a    float64
b    Float64
dtype: object
```